esx_service/utils/auth_data.py

```diff
@@ -665,7 +690,20 @@ def _remove_volumes_for_tenant(self, tenant_id):
                     logging.error("remove vmdk %s failed with error %s", vmdk_path, err)
                     error_info += err
             
-            # Delete path /vmfs/volumes/datastore_name/tenant_name
+            # delete the symlink /vmfs/volume/datastore_name/tenant_name
+            # which point to /vmfs/volumes/datastore_name/tenant_uuid
+            for (datastore, url_name, path) in vmdk_utils.get_datastores():
+                dock_vol_path = os.path.join("/vmfs/volumes", datastore, vmdk_ops.DOCK_VOLS_DIR)
+                tenant_path = os.path.join(dock_vol_path, tenant_id)
+                logging.debug("remove volumes for tenant: try to remove the symlink to path %s", tenant_path)
```

Comment:
message is a bit unclear. Maybe "Tenant <name> removal: removing symlink <name>" ?
And the rest of the messages should be similar. Also please avoid copy-n-paste for logging format, 
instead use something like 
```python
VOL_RM_LOG_PREFIX = ""Tenant <name> removal: "

 logging.debug(VOL_RM_LOG_PREFIX + "Removing  symlink to  %s", tenant_id, tenant_path)
```
(or any other reusabe code)